### PR TITLE
fix(api): remove `gaps` endpoint

### PR DESCRIPTION
**Pull request by grtlr**

### CODE REVIEW

**Comment by grtlr** [comment]
Don't merge yet.

**Review by Alexandcoats — APPROVED** [review]

File: bin/inx-chronicle/src/api/stardust/core/routes.rs [hunk 1]
```diff
@@ -91,6 +97,80 @@ pub fn routes() -> Router {
...
+                .get_milestone_id(newest_milestone.milestone_index)
+                .await?
+                .ok_or(ApiError::Internal(InternalApiError::CorruptState(
+                    "no milestone in the database",
+                )))?,
+        )
+        .to_string(),
+    };
+
+    // Unfortunately, there is a distinction between `LatestMilestoneResonse` and `ConfirmedMilestoneResponse` in Bee.
```
Comment (Alexandcoats) [inline bin/inx-chronicle/src/api/stardust/core/routes.rs:143]: typo `LatestMilestoneResonse`

File: src/db/collections/milestone.rs [hunk 2]
```diff
@@ -143,22 +146,26 @@ impl MongoDb {
             .transpose()?)
     }
 
-    /// Gets the timestamp of a milestone by the [`MilestoneIndex`].
-    pub async fn get_milestone_timestamp(&self, index: MilestoneIndex) -> Result<Option<MilestoneTimestamp>, Error> {
```
Comment (Alexandcoats) [inline src/db/collections/milestone.rs:147]: Why remove this?

Comment (grtlr) [reply src/db/collections/milestone.rs:147]: We never called it, so I think it's save to delete.

**Pull request merged**